### DISCUSSION TITLE
The current and the previous request authorization don't match

### DISCUSSION THREAD

**Discussion by sabikrahat** [pocketbase/pocketbase#2686] [Q&A] [answered]
My Application runs almost all collections subscriptions in the background. So I've unsubscribed all the collections during the time of logging out.

`
  await pb.collection(users).unsubscribe('*');
  await pb.collection(orders).unsubscribe('*');
  await pb.collection(vendors).unsubscribe('*');
  await pb.collection(employees).unsubscribe('*');
  await pb.collection(inventories).unsubscribe('*');
  await pb.collection(transactions).unsubscribe('*');
  pb.authStore.clear();
`

It is working fine. but whenever I log in again with out restarting the application it shows an error like that 

`
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: ClientException: {url: http://127.0.0.1:8090/api/realtime, isAbort: false, statusCode: 403, response: {code: 403, message: The current and the previous request authorization don't match., data: {}}, originalError: null}
`

If anyone goes throw this situation please let me know how you bypass this warning. Though it is working after login again. But shows the warning and I want to fix this. 

Thank you.

**Comment by ganigeorgiev** [comment, marked as answer]
Most likely you have an old active realtime subscription/connection before signing in with the new user and submitting the new subscriptions.

In your logout action you can unsubscribe from all topics at once with `pb.realtime.unsubscribe()` (_I'll have to think a little more on it, but maybe we can eventually call it automatically behind the scenes on store model change_).

**Comment by sabikrahat** [comment]
after executing this while logging out `pb.realtime.unsubscribe()` the warning is not showing. Thank you so much @ganigeorgiev. 

I think it'll be better if it automatically executes on store model change.
